pkg/doc/did/helpers.go

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package did
+
+import (
+	"fmt"
+)
+
+// LookupService returns the service from the given DIDDoc matching the given service type.
+func LookupService(didDoc *Doc, serviceType string) (*Service, error) {
```

Comment:
did you intend to have lookup functions with an error return instead of bool?

Reply:
I'll make them return bool today.

Reply:
done